### PR TITLE
Add Python3 support

### PR DESCRIPTION
**Pull request by mauriciosl**
This will add some fixes to support both python3.4 and python2.7, the tests are passing and i have been using it so far without bugs.
Maybe launch it in a new major release?

thanks

### CODE REVIEW

File: test/test_stringconversions.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class StringConversionsTest(unittest2.TestCase):
         'dict': {
             'decimal': Decimal('1.4'),
         },
-        'list': [Decimal('1.0'), datetime.date(2000, 01, 01)],
+        'list': [Decimal('1.0'), datetime.date(2000, 1, 1)],
```
Comment (czue) [inline test/test_stringconversions.py]: nice catch :)

**Comment by czue** [comment]
@mauriciosl this looks great to me. @dannyroberts maintains this library and is on vacation so I'll wait for him to come back and review/merge.

thanks for the contributions!

**Comment by mauriciosl** [comment]
No problem @czue, i loved the library and am implementing some more new features, when he's back we talk about it.

cya

**Comment by dannyroberts** [comment]
Thanks, @mauriciosl! This looks great.

**Comment by dannyroberts** [comment]
@mauriciosl happy to hear your thoughts on new features! I'm interested in keeping this library's interface small, so I'll probably ask questions like "Could you implement the same thing as functionality outside the library?" "Is this feature broadly useful in nature, or overly specific to be included?" etc.

**Comment by mauriciosl** [comment]
Hi @dannyroberts, looks good to me. The feature i missed in the library was serializing the objects back to json. I'm using the library to parse incoming api data into objects that can do more than the simple python data structures, and sometimes they need to be serialized back into json, JsonArray and other containers for example lacks this feature.

**Comment by dannyroberts** [comment]
I think you're looking for obj.to_json()

**Pull request merged**